### ISSUE TITLE
hava function like  "open bookmark" of vimium

### ISSUE THREAD

**Issue by zcjava** [televator-apps/vimari#249]
I use chrome's vimium, I can use "Shift+b" open bookmark. vimari hava the function?

**Comment by moeyua** [comment]
I really, really need this feature, it's pretty much the one I use most often.

**Comment by kruecab** [comment]
Agreed! Trying to move from Firefox+Vimium to Safari and this is something I rely on.

**Comment by fernanluyano** [comment]
I am actually switching to Firefox because of this
